src/views/GameRoom/components/MyHand/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { DeckWrapper, HandCardsWrapper, HandCardWrapper } from './styles';
+import { PlayingCard } from '../../../../components/PlayingCard';
+
+const cards = Array(15).fill({
```

Comment:
We should think about not render all the cards in the dom and try a way to render only one card and simulate the rest

Reply:
But this can be done later when adding the events listeners